src/main/scala/sbtbuildinfo/ScalaFinalCaseObjectRenderer.scala

```diff
@@ -0,0 +1,88 @@
+package sbtbuildinfo
+
+private[sbtbuildinfo] case class ScalaFinalCaseObjectRenderer(options: Seq[BuildInfoOption], pkg: String, obj: String) extends ScalaRenderer {
```

Comment:
Could you add scripted test to see if this can be used? I don't know if you can with package private.

Reply:
Whoops, you're right!

Reply:
scripted test please - https://github.com/sbt/sbt-buildinfo/tree/master/src/sbt-test/sbt-buildinfo/caseclassrenderer
https://www.scala-sbt.org/1.x/docs/Testing-sbt-plugins.html

Reply:
Does this look good?

Reply:
Awesome. Thanks!